README.md

```diff
@@ -0,0 +1,45 @@
+# GitHub Email Webhook
+
+This package provides a Python Lambda function which handles GitHub
+webhooks. It creates and sends an email to the provided mailing list/email.
+The reason for the function is to replace the GitHub email service which is
+now deprecated.
+
+## Requirements
+
+- boto3
+
+## Install
+
+Prior to deployment ensure that you have boto3 installed. If boto3 is
+configured with a default account you will not be required to input the
+access key id and secret access key during deployment. For more information
+see the [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#shared-credentials-file). 
+
+To install the function and REST API call: `python deploy.py`.
+
+Once the function is installed a secret key and REST API endpoint will
+be provided. Use these two values to configure your GitHub repo's webhook.
+
+For more information see the
+[GitHub docs](https://developer.github.com/webhooks/creating/).
+
```

Comment:
Should mention that kms needs to be enabled in account

Reply:
I don't think it's a required step. With this setup we are using two keys. For the variables the default key is used for encryption in storage. The key which is created during the script is used as second layer of encryption for the variables in transit.

I tested this in a region where I had not used KMS and create_function creates the default key without issue.

Is there something at the account level I'm missing? It's not mentioned in docs if so: https://docs.aws.amazon.com/kms/latest/developerguide/getting-started.html.